_data/monthly-calls.yml

```diff
@@ -1,4 +1,23 @@
 videos:
+  - title: Component-based accessibility tests for the U.S. Web Design System
+    subtitle: Manual accessibility tests for the whole team
+    description: |
+      We shared new component-based accessibility test pages (formerly called “critical checklists”) for 
+      the [accordion](https://designsystem.digital.gov/components/accordion/accessibility-tests/), 
+      [button](https://designsystem.digital.gov/components/button/accessibility-tests/), and 
+      [link](https://designsystem.digital.gov/components/link/) 
```

Comment:
This link is different from the other two. Should the link go to the accessibility test page?

Reply:
Thanks, Katie, I've fixed the link.